pkg/storage/swift/swift_test.go

```diff
@@ -581,6 +582,13 @@ func TestSwiftEndpointTypeObjectStore(t *testing.T) {
 	th.AssertEquals(t, container, installConfig.Status.Storage.Swift.Container)
 }
 
-func TestSwiftGenerateContainerName(t *testing.T) {
-	th.AssertEquals(t, false, generateContainerName("image_registry_") == generateContainerName("image_registry_"))
+func TestSwiftGetClusterID(t *testing.T) {
```

Comment:
Not sure you really need this test? The clusterID should be provided by the infrastructures config.

Reply:
Yep, I removed this test